control/statesp.py

```diff
@@ -54,7 +54,7 @@
 import math
 import numpy as np
 from numpy import all, angle, any, array, asarray, concatenate, cos, delete, \
-    dot, empty, exp, eye, matrix, ones, poly, poly1d, roots, shape, sin, \
+    dot, empty, exp, eye, matrix, ones, pi, poly, poly1d, roots, shape, sin, \
```

Comment:
We should avoid using `numpy.pi` since this can generate errors when `numpy` is incorporated as a `mock` function.  See [PR 171, commit b881b7](https://github.com/python-control/python-control/pull/171/files/b881b77e485be3c7d90a56a0db8d190b56cdf8c1).  Replace with `math.pi`.

Reply:
unqualified `pi` no longer used following your aa6e0dfc anyway.  Will remove.